FirebaseRemoteConfig.podspec

```diff
@@ -68,6 +68,6 @@ app update.
         'FirebaseRemoteConfig/Tests/Unit/Defaults-testInfo.plist',
         'FirebaseRemoteConfig/Tests/Unit/SecondApp-GoogleService-Info.plist'
     unit_tests.requires_app_host = true
-    unit_tests.dependency 'OCMock', '~> 3.4.0'
+    unit_tests.dependency 'OCMock', '~> 3.5.0'
```

Comment:
Consider no version here and letting the dependency float like the podspecs.

If something breaks when OCMock updates, it doesn't impact the product and we probably want to know about it.

Reply:
sg. Done.